src/layouts/_components/UserMenu.tsx

```diff
@@ -3,13 +3,23 @@ import cn from "classnames";
 import Link from "~/components/Link";
 import OutsideClick from "~/components/OutsideClick";
 
-const menuItems = [
+interface SubItem {
+  to: string;
+  text: string;
+}
+
+interface MenuItem {
+  title: string;
+  children: Array<SubItem>;
+}
+
+const menuItems: Array<MenuItem> = [
```

Comment:
@ivanlori, FYI, I've changed this part and added proper types to avoid using `any` 🚀  Thanks for the PR!